app/Resources/views/base.html.twig

```diff
@@ -7,16 +7,28 @@
         {% block stylesheets %}
             <link href="{{ asset('bundles/oag/stylesheets/normalize.css') }}" rel="stylesheet" />
```

Comment:
This needs to be replaced with the new method of including stylesheets (through assetic) see https://github.com/NeonOpenAG/oagtoolbox/blob/9ceda088f8d31a16018cdcfcad4c0a2e35dd1914/src/OagBundle/Resources/views/Default/index.html.twig#L6 for an example.